experimental/split_mlir/src/iree/split_mlir/MarkBisectPassImpl.h.inc

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 The IREE Authors
```

Comment:
Is there a reason we have to commit a .inc file?

Reply:
Build time is already a pain as there are many translation units. Usually most of the time is spent parsing the same header files. I want one CPP but I also want to split the code in multiple files. The file is not meant to be included by the user. I can make it just a `.h`.